### DISCUSSION TITLE
(sRGB) EPS to CMYK TIFF file

### DISCUSSION THREAD

**Discussion by tk-pxx** [ImageMagick/ImageMagick#2449] [Help] [answered]
Hello,
I am trying to convert an EPS file (sRGB) to a TIFF file (CMYK).
The output colors of the CMYK TIFF are really off, they almost look inverted.
This happens with all EPS files I tested.
Edit: The same happens also for AI/SVG/PDF files (so in fact all common vector graphics) and also when the input vector graphic file is in CMYK already.

ImageMagick Version: ImageMagick 7.0.8-61 Q16 x86_64 2019-08-19
System: Debian 4.9.228-1
Ghostscript: GPL Ghostscript 9.26 

My ImageMagick call:
`magick input.eps -profile mysRGB.icc -profile myCMYK.icc output.tiff`

same happens with this call:
`magick input.eps -colorspace CMYK output.tiff`

The problem also exists for other common vector grahpic filetypes (AI/SVG/PDF) as well as when I first convert the file to an sRGB TIFF/PNG file (which looks perfectly fine) and then convert this sRGB TIFF/PNG to a CMYK TIFF.

Example input EPS files, the according output cmyk TIFFs, as well as a PNG file (which was converted from an EPS file) where this bug happens are located in the zip file:
[exampleFiles.zip](https://github.com/ImageMagick/ImageMagick/files/5108465/exampleFiles.zip)

Any suggestions on how to fix this problem?
Thank you in advance.

**Comment by snibgo** [comment, marked as answer]
"-profile sRGB.icc -profile USWebUncoated.icc" works fine for me, and doesn't invert colours. For my test images they look fine when displayed in Microsoft Photo Viewer. Perhaps your viewer can't handle CMYK images.

Your PNG file is RGB, not CMYK. The PNG format canot record CMYK.

Your images are graphics, and I don't know what they should look like. I suggest you include "should.png" in your zip.

**Reply by tk-pxx** [reply]
You are totally right, thank you! I was using IfranView which showed the colors wrong, as well as photopea which also fails at displaying the TIFF images correctly. Sorry for wasting your time, the solution should have been pretty obvious.
